packages/core/src/Asset/NftMetadata/index.ts

```diff
@@ -0,0 +1,11 @@
+import { fromMetadatum } from './fromMetadatum';
+
+import { NftMetadata as _NftMetadata } from './types';
+
+export * from './types';
+
+// This seems to only be working when both type and value are defined in the same file
```

Comment:
What "seems" to be working?

Reply:
I couldn't
`export { NftMetadata } from './types'`
+
`export const NftMetadata = {`,
but it works as expected (can use as type or as value) when exporting both from the same file

Reply:
```suggestion
// tsc requires the type and object, sharing a name, to be exported from the same module
```

Reply:
Thanks! 933ac784f612c15daa46184640c92f1befb66ff6